mta/entities/projects.py

```diff
@@ -57,7 +57,7 @@ class add_applications(View):  # noqa
         delete_application = Text(locator=".//button[contains(@aria-label, 'delete-application')]")
         browse_button = Button("Browse")
         upload_file = HiddenFileInput(
-            locator='.//input[@accept=".ear,.har,.jar,.rar,.sar,.war,.zip"]'
+            locator='.//input[@accept=".ear, .har, .jar, .rar, .sar, .war, .zip"]'
         )
```

Comment:
I checked locator on 5.1.0.Final version. We don't need to update it. Old locator is correct. please check once again.

Reply:
Yes the locator is changed in latest snapshot . So let's keep it the way it was